include/charstream.h

```diff
@@ -8,8 +8,12 @@
  * Transform input from a file descriptor into a stream of characters.
  */
 
+#define CHARSTREAM_EOF EOF
```

Comment:
```suggestion
#define CHARSTREAM_EOF (EOF)
```
EOF expands to a numerical value so it's always nice to have parentheses around those

Reply:
In the case of glibc EOF expands to ``(-1)`` [see](https://github.com/lattera/glibc/blob/895ef79e04a953cac1493863bcae29ad85657ee1/libio/stdio.h#L104)
But yes valid point